config/routes.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+Rails.application.routes.draw do
+  get '/categories/:category_id/payment' => 'payments#index', as: :category_payments
+  get '/categories/:category_id/payment/new' => 'payments#new', as: :new_payment
+  post '/categories/:category_id/payment' => 'payments#create', as: :create_payment
```

Comment:
[REQUIRED] The endpoints you have here have already been created by the resources command in line 9 of this routes file. Either use `only` or `except` to exclude these routes from being created by resources if you want to keep the names you are adding with `as:` or remove these lines and stick to using resources with the names that are automatically generated for you